lib/anyway/config.rb

```diff
@@ -127,11 +130,41 @@ def config_attributes
       end
 
       def required(*names)
-        unless (unknown_names = (names - config_attributes)).empty?
-          raise ArgumentError, "Unknown config param: #{unknown_names.join(",")}"
+        unknown_names = names - config_attributes
+        env_option = unknown_names.find { |name| env_attr?(name) }
```

Comment:
Why can't we use a keyword argument here (`def required(*names, env: nil)`)?
Extracting Hash from an array adds a lot of complexity.